Examples/Swift/InsetMapExample.swift

```diff
@@ -0,0 +1,61 @@
+import Mapbox
+
+@objc(InsetMapExample_Swift)
+
+class InsetMapExample_Swift: UIViewController, MGLMapViewDelegate {
+    var mapView: MGLMapView!
+    var miniMapview: MGLMapView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        mapView = MGLMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.delegate = self
+        
+        // Set the main map's center coordinate and zoom level.
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 18.1096, longitude: -77.2975), zoomLevel: 9, animated: false)
+
+        // Set inset mapView's center
+        miniMapview = MGLMapView(frame: CGRect.zero)
+        
+        // Set inset mapview properties to create a smaller, non-interactive mapView that mimics the appearance of the main mapView.
+        miniMapview.allowsScrolling = false
+        miniMapview.allowsTilting = false
+        miniMapview.allowsZooming = false
+        miniMapview.allowsRotating = false
+        miniMapview.compassView.isHidden = true
+        
+        // we are only hiding the attribution within the second map view. Ordinarily, hiding the map view's attribution would be a violation of TOS, however, because the main map view still has its attribution, there is no issue here.
```

Comment:
- `we` → `We`
- Please wrap this into a multi-line comment limited to 80 chars per line as mentioned earlier.
- Avoid using acronyms per our internal style guide. An edit suggestion:

```
/**
 Hiding the map view's attribution goes against our attribution requirements. However, because the main 
 map view still has attribution, hiding the attribution for the mini map view in this case is acceptable. 

 For more information, please refer to our attribution guidelines: https://docs.mapbox.com/help/how- mapbox-works/attribution/
*/
```